src/hotspot/share/classfile/stringTable.cpp

```diff
@@ -888,7 +888,7 @@ oop StringTable::init_shared_table(const DumpedInternedStrings* dumped_interned_
   verify_secondary_array_index_bits();
 
   _shared_table.reset();
-  CompactHashtableWriter writer(_items_count, ArchiveBuilder::string_stats());
+  CompactHashtableWriter writer((int)_items_count, ArchiveBuilder::string_stats());
```

Comment:
This makes me question why `Items_count` Is a size_t, when we treat it as an int.

Reply:
Good question. I see no reason why it needs to be.  Also _items_count in StringTable is the same.